From bd177c781e404b61a7f56fa522345b34f4a75735 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Mon, 8 Sep 2014 13:18:32 -0400
Subject: [ethernet] r8169: add missing MODULE_FIRMWARE

Message-id: <1410182312-19560-14-git-send-email-ivecera@redhat.com>
Patchwork-id: 91991
O-Subject: [RHEL7.1 PATCH 13/13] r8169: add missing MODULE_FIRMWARE.
Bugzilla: 1139247
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: John Linville <linville@redhat.com>

BZ: #1139247

Upstream commit(s):
commit a3bf5c429eb5f5ec4d364d51dfa8855efcc005f8
Author: Francois Romieu <romieu@fr.zoreil.com>
Date:   Tue Aug 26 22:40:38 2014 +0200

    r8169: add missing MODULE_FIRMWARE.

    Leftover from 6e1d0b8988188956dac091441c1492a79a342666 ("r8169:add
    support for RTL8168H and RTL8107E").

    Signed-off-by: Francois Romieu <romieu@fr.zoreil.com>
    Cc: Chun-Hao Lin <hau@realtek.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/realtek/r8169.c b/drivers/net/ethernet/realtek/r8169.c
index 22dfa8c..30217c1 100644
--- a/drivers/net/ethernet/realtek/r8169.c
+++ b/drivers/net/ethernet/realtek/r8169.c
@@ -852,6 +852,8 @@ MODULE_FIRMWARE(FIRMWARE_8168G_2);
 MODULE_FIRMWARE(FIRMWARE_8168G_3);
 MODULE_FIRMWARE(FIRMWARE_8168H_1);
 MODULE_FIRMWARE(FIRMWARE_8168H_2);
+MODULE_FIRMWARE(FIRMWARE_8107E_1);
+MODULE_FIRMWARE(FIRMWARE_8107E_2);
 
 static void rtl_lock_work(struct rtl8169_private *tp)
 {
-- 
1.7.1